scripts/docker/Dockerfile

```diff
@@ -0,0 +1,61 @@
+FROM golang:1.20 as builder
+
+# Download monero-wallet-rpc. We need bzip2 to unpack the tar file.
+RUN apt update && apt install -y bzip2
+RUN arch=$(uname -m | sed 's/x86_64/linux64/; s/aarch64/linuxarm8/') && \
+    curl -sSL "https://downloads.getmonero.org/cli/${arch}" -o monero.tar.bz2
+RUN tar xvjf monero.tar.bz2 --no-anchored monero-wallet-rpc --strip-components=1
+
+# Build the swapd and swapcli binaries. The BRANCH argument can be set to a
+# branch, release tag, "latest", or a commit hash.
+ARG VERSION=master
```

Comment:
I wanted to use `latest` here, but it was causing issues because we haven't used a release tag in recent history. Note: Branch names can't have a `/` in them, which most of our branches do.

Reply:
let's be sure to update this when we tag a release